apis/github/code/src/tools.js

```diff
@@ -0,0 +1,214 @@
+export async function closeIssue(octokit, owner, repo, issueNumber) {
+    try {
+        await octokit.issues.update({
+            owner,
+            repo,
+            issue_number: issueNumber,
+            state: 'closed'
+        });
+        console.log(`Closed issue #${issueNumber} - https://github.com/${owner}/${repo}/issues/${issueNumber}`);
+    } catch (error) {
+        console.error(`Error closing issue: ${error}`);
```

Comment:
This and the other `console.error` calls should be changed to `console.log`

Reply:
refactored to log any errors that get thrown at the top-level